compiler/src/iree/compiler/Codegen/Common/test/fission_transfer_ops_control_flow.mlir

```diff
@@ -109,3 +109,22 @@ func.func @fission_unit_trip(%arg0: memref<1x?x?x8xbf16, #amdgpu.address_space<f
   return
 }
 // CHECK-ALL-COUNT-2: scf.for
+
+// -----
```

Comment:
Could you add a unit test with 2 transfer_read/write pairs and make sure it fail? Like the example we've encountered in the bug.